app/controllers/beers_controller.rb

```diff
@@ -4,7 +4,7 @@ class BeersController < ApplicationController
 
   def index
     @beers = if params[:search]
-      Beer.where('name LIKE ?', "%#{params[:search]}%").order(:name).page(params[:page])
```

Comment:
if is posible, try to use no more than 80 characters per line, (you have 104).
```ruby
      Beer.where('lower(name) LIKE ?', "%#{params[:search].downcase}%").order(:name).page(params[:page])
```
what do you think about this?
```ruby
      Beer.where('lower(name) LIKE ?', "%#{params[:search].downcase}%").
        order(:name).page(params[:page])
```

Reply:
Ok, but I prefer with the point in the second line

```      
Beer.where('lower(name) LIKE ?', "%#{params[:search].downcase}%")
  .order(:name).page(params[:page])
```

Reply:
@jnegrete done!